extras/courses.md

```diff
@@ -42,6 +51,8 @@ Courses | Duration | Effort
 
 Courses | Duration | Effort
 :-- | :--: | :--:
+[Data Structures and Algorithms specialization](https://www.coursera.org/specializations/data-structures-algorithms) | 6-8 months | 4-8 hours/weeks
+[Algorithms specialization](https://www.coursera.org/specializations/algorithms) | 16 weeks | 4-8 hours/weeks
```

Comment:
Stanford Algorithms, which you've linked here, is actually identical to the "Algorithms: Design and Analysis" courses that we require as part of the official curriculum. It's the same course under a different name and on a different platform. So you can delete this line.